lib/xcodeproj/project/object/build_phase.rb

```diff
@@ -279,16 +279,14 @@ class PBXShellScriptBuildPhase < AbstractBuildPhase
 
         # @return [String] the actual script to perform.
         #
-        # @note   Defaults to the empty string.
+        # @note   Defaults to a comment string.
         #
-        attribute :shell_script, String, ''
+        attribute :shell_script, String, "# Type a script or drag a script file from your workspace to insert its path.\n"
 
         # @return [String] whether or not the ENV variables should be shown in
         #         the build log.
         #
-        # @note   Defaults to true (`1`).
-        #
-        attribute :show_env_vars_in_log, String, '1'
+        attribute :show_env_vars_in_log, String
```

Comment:
Should something change such that setting this to '1' is equivalent to clearing it?

Reply:
I am not sure we want to add this logic here so much....I am making the PR in CocoaPods to ensure that this does not happen though